notebooks/txpipe_cluster_background_selection.ipynb

```diff
@@ -7,20 +7,28 @@
    "source": [
     "# TXPipe - CLMM Data Preparation\n",
     "\n",
-    "This notebook runs and explores two pipelines that generate the weighted, calibrated, per-cluster background shear catalogs as inputs to CLMM.\n",
+    "This notebook runs and explores two pipelines that generate the weighted, calibrated, per-cluster background shear catalogs as inputs to CLMM.  The instructions for running this on IN2P3 and on NERSC will differ.\n",
     "\n",
-    "Before starting you will need to:\n",
+    "On IN2P3 -- before starting you will need to:\n",
     "- set up the TXPipe environment at CC-IN2p3 using the command \n",
     "`source /pbs/throng/lsst/users/jzuntz/txpipe-environments/setup-txpipe`\n",
     "- clone the TXPipe repository somewhere, preferably in the SPS space since we will make some large files.\n",
```

Comment:
Maybe remove the statement about cloning in the sps folder (the recommendation is for big data files to live there, but not code I believe)